recipes/diann/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "1.8.1" %}
+{% set sha256 = "2c38201b5e04e9e0df621ba053bbad93d70dc9b23bd6e0b78a1f6b22315cc1c0" %}
+
+package:
+  name: diann
+  version: {{ version }}
+
+source:
+  - url: https://github.com/vdemichev/DiaNN/releases/download/{{ version }}/diann_{{ version }}.deb
+    sha256: {{ sha256 }}
+
+build:
+  number: 0
+  binary_relocation: False
+  noarch: generic
+
+requirements:
+  build:
+  host:
+    - glib >=2.27
+  run:
+    - glib >=2.27
+
+test:
+  commands:
+    - diann-{{ version }}
+
+about:
+  home: https://github.com/vdemichev/DiaNN
+  license: https://github.com/vdemichev/DiaNN/blob/master/LICENSE.txt
```

Comment:
this is the `license_file` not the license.
Its not clear which type of license this is :(

Reply:
ty, I'll change it. Do you think the license does not allow use as a Bioconda package? It sounded like it would be okay.

Reply:
I don't know, it's about IP and one single copy as a backup... Do you know this group?

Reply:
I'll ask @vdemichev